semgrep_output_v1.atd

```diff
@@ -632,6 +647,10 @@ type cli_output_extra <ocaml attr="deriving show"> = {
      *)
     (* EXPERIMENTAL: since semgrep 0.109? *)
     ?explanations: matching_explanation list option;
+
+    (* since: 1.10.0 *)
```

Comment:
Maybe you can add an EXPERIMENTAL, so people know those fields could change and
they should not rely on it.